command/command.ts

```diff
@@ -1766,9 +1780,17 @@ export class Command<
         }
       }
 
-      return await this.execute(options, args);
+      if (ctx.execute) {
+        return await this.execute(options, args);
+      } else {
+        return { options, args, cmd: this, literal: this.literalArgs };
+      }
     } catch (error: unknown) {
-      this.handleError(error);
+      if (ctx.execute) {
```

Comment:
I wasn't sure about this, but I don't think we want the help execution/etc. run and displayed on the command line when the user has provided `execute: false`.